patches/minecraft/net/minecraft/block/BlockBush.java.patch

```diff
@@ -14,19 +14,7 @@
     public boolean func_196260_a(IBlockState p_196260_1_, IWorldReaderBase p_196260_2_, BlockPos p_196260_3_) {
        BlockPos blockpos = p_196260_3_.func_177977_b();
 +      if (p_196260_1_.func_177230_c() == this) //Forge: This function is called during world gen and placement, before this block is set, so if we are not 'here' then assume it's the pre-check.
-+         return p_196260_2_.func_180495_p(blockpos).canSustainPlant(p_196260_2_, blockpos, EnumFacing.UP, this);
++         return p_196260_2_.func_180495_p(blockpos).canSustainPlant(p_196260_2_, blockpos, EnumFacing.UP, p_196260_1_, this);
```

Comment:
The point was that in some cases {simulation/pre-checking} the plant's blockstate is not available. So... Why are you passing it in?

Reply:
I see no reason the block trying to sustain can't be given a possible state, that state doesn't *have* to be in the world, that's the point of it being a parameter.